### PR TITLE
Allow multiple cachetool calls

### PR DESCRIPTION
**Pull request by infabo**
- [x] Bug fix #…? No
- [x] New feature? Yes
- [x] BC breaks? I would say no. Possibly breaking for people that modified the recipe's tasks.
- [x] Tests added? No tests I am aware of. So none adjusted.
- [x] Docs added? Docs regenerated

      Please, regenerate docs by running next command:
      $ php bin/docgen

### CODE REVIEW

File: contrib/cachetool.php [hunk 1]
```diff
@@ -64,36 +66,51 @@
...
     $options = get('cachetool');
     $fullOptions = get('cachetool_args');
+    $return = [];
 
     if (strlen($fullOptions) > 0) {
-        $options = "{$fullOptions}";
+        $return = ["{$fullOptions}"];
     } elseif (strlen($options) > 0) {
-        $options = "--fcgi={$options}";
+        if (!is_array($options)) {
```
Comment (Schrank) [inline contrib/cachetool.php:74]: You can shorten that by just casting it to array - if it is a string it becomes ['string'], if it is array it stays as it is :-)

File: contrib/cachetool.php [hunk 2]
```diff
@@ -64,36 +66,51 @@
 set('cachetool_options', function () {
     $options = get('cachetool');
     $fullOptions = get('cachetool_args');
+    $return = [];
 
     if (strlen($fullOptions) > 0) {
-        $options = "{$fullOptions}";
+        $return = ["{$fullOptions}"];
```
Comment (Schrank) [inline contrib/cachetool.php:72]: Is there a reason for "{ because imho you can remove it and just have `[$fullOptions]`

**Review by Schrank — CHANGES_REQUESTED** [review]
That looks great! Interesting idea :-)

File: contrib/cachetool.php (hunk 2 above)
Comment (infabo) [reply contrib/cachetool.php]: It was there before. Did not change it. But yes, it is not needed.

Comment (Schrank) [reply contrib/cachetool.php]: Thanks for the cleanup :-)

**Review by Schrank — APPROVED** [review]
Looks good to me!

**Comment by yard-mschwartz** [comment]
It turns out we had a copy of the default opcache clear command in `deploy.php`, which broke the our deployment pipeline since v7.3.2, that includes this change. Only God knows why. :upside_down_face: 

The error being: Array to string conversion in Configuration.php on line 136

Anyway, this is now solved by removing this copy, that shouldn't have been there in the first place.

**Pull request merged**